src/docs/references/api-reference/index.md

```diff
@@ -17,7 +17,7 @@ The Sharetribe Flex API consists of three individual APIs:
 - [Integration API](https://www.sharetribe.com/api-reference/integration.html)
 - [Authentication API](https://www.sharetribe.com/api-reference/authentication.html)
 
-All three are HTTP APIs. The Marketplace API and the Integration API support both JSON and Transit formats. The Authentication API is used to obtain access tokens for use with the other APIs and is based on OAuth2. The easiest way the access all three APIs is to use our [JavaScript SDKs](https://sharetribe.github.io/flex-sdk-js/).
+All three are HTTP APIs and both the Integration API and Marketplace API support JSON and Transit formats. The Authentication API is based on OAuth2. Our API documentation provides further technical detail on all three APIs. If you are looking for a way to use the APIs in your client or server application, refer to our documentation on our [JavaScript SDKs.](https://sharetribe.github.io/flex-sdk-js/)
```

Comment:
```suggestion
All three are HTTP APIs and both the Integration API and Marketplace API support JSON and Transit formats. The Authentication API is based on OAuth2. Our API documentation provides further technical detail on all three APIs. If you are looking for a way to use the APIs in your client or server application, refer to our documentation on our [JavaScript SDKs](https://sharetribe.github.io/flex-sdk-js/).
```

Reply:
We could actually differentiate here between the SDK docs links as this link here only points to the JS SDK documentation and https://sharetribe.github.io/flex-integration-sdk-js/. So there could be two links here.